docker_shell.sh

```diff
@@ -44,6 +44,9 @@ fi
 export DESIGN_CONFIG_PREFIXED=$WORKSPACE_EXECROOT/$DESIGN_CONFIG
 export STAGE_CONFIG_PREFIXED=$WORKSPACE_EXECROOT/$STAGE_CONFIG
 
+# Make bazel-bin writable
+chmod -R +w $WORKSPACE_EXECROOT/bazel-out/k8-fastbuild/bin
```

Comment:
It is a bit unnerving that files are overwritten in this folder. I something begin rebuilt that shouldn't?